pkg/byteconv/byteconv.go

```diff
@@ -67,5 +67,5 @@ func ParseUint64(b []byte) (uint64, error) {
 }
 
 func ParseFloat64(b []byte) (float64, error) {
-	return strconv.ParseFloat(UnsafeString(b), 10)
+	return strconv.ParseFloat(UnsafeString(b), 64)
```

Comment:
In case anyone else wondered: I looked at the strconv.ParseFloat source; unless the bitsize param is 32, it will parse as 64 bit.